api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -290,22 +290,16 @@ async function initPatientDocQuery(
   totalDocs: number,
   convertibleDocs: number
 ): Promise<Patient> {
-  return updateDocQuery({
+  return appendDocQueryProgress({
     patient: { id: patient.id, cxId: patient.cxId },
     downloadProgress: {
       status: "processing",
       total: totalDocs,
     },
-    convertProgress:
-      convertibleDocs > 0
-        ? {
-            status: "processing",
-            total: convertibleDocs,
-          }
-        : {
-            status: "completed",
-            total: 0,
-          },
+    convertProgress: {
+      status: "processing",
+      total: convertibleDocs,
+    },
```

Comment:
Start the process as processing, let the definition about `completed` be done after the initial processing/download